packages/backend/src/db/db.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { MongooseModule } from '@nestjs/mongoose';
+import { SampleService } from './sample/sample.service';
+import { Sample, SampleSchema } from './sample/sample.schema';
+
+@Module({
+  imports: [
+    MongooseModule.forFeature([{ name: Sample.name, schema: SampleSchema }]),
+  ],
+  providers: [SampleService],
+})
```

Comment:
Does SampleService need to be exported for use in other modules?

Reply:
My bad on this taking so long, busy weekend. As we discussed before We put all the store services under the db module and then each service the requires a store can import the db module and then use any store service it desires.

I'm facing an issue with the tests that i'm hoping to resolve tonight with Jackson. You could also help me if you have time

Reply:
@wqsz7xn in the meantime make sure it works with the api (disregarding the tests)